wip-0016.md

```diff
@@ -187,7 +187,10 @@ A reference implementation for the proposed protocol improvements labeled as  **
 
 ## Adoption Plan
 
-An adoption plan will be proposed upon moving this document from the _Draft_ stage to the _Proposed_ stage.
+An activation date is proposed for July 13 2021 at 9am UTC, that is, protocol epoch #522240.
+
+From the perspective of TAPI, this proposal will be signaled along with the WIP-0014. The `first_signaling_epoch` for the base implicit signal (
```

Comment:
```suggestion
From the perspective of TAPI, this proposal will be signaled along with the [WIP-0014]. The `first_signaling_epoch` for the base implicit signal (
```

Reply:
The _base implicit signal_ shouldn't be "set" here, as it was already set in WIP-0014. In this case, what we can say is that this proposal adheres to the _base implicit signal_, and that the _first signaling epoch_ is also set to that of WIP-0014.